### ISSUE TITLE
module Ewmh/xworkspaces is not showing in polybar

### ISSUE THREAD

**Issue by arcolinuxz** [hyprwm/Hypr#91]
**Please describe the bug**

we do not see the workspaces in our polybar 
should be in the middle top bar

https://github.com/arcolinux/arcolinux-hypr/tree/main/etc/skel/.config/hypr/polybar

**Steps to reproduce:**
Install our app package on ArcoLinux

Package is renamed to hypr-dev-git (same from AUR - named there hypr-git added xorg-xmodmap as dependency) 
Using a different naming convention to keep track of your commits to github
r369.3e3d943-2 - commit 369 at the moment
https://github.com/arcolinux/arcolinux_repo_3party/blob/main/x86_64/hypr-dev-git-r369.3e3d943-2-x86_64.pkg.tar.zst

Install our config package on ArcoLinux

https://github.com/arcolinux/arcolinux_repo/blob/main/x86_64/arcolinux-hypr-git-22.11-06-any.pkg.tar.zst
https://github.com/arcolinux/arcolinux-hypr

**Expected behavior**
We would like to see multiple workspaces.

**Video**
3 issues

1. keybindings (quit application for example)
2. ewmh on polybar
3. system panel icons not responding on a click

https://youtu.be/s84yLa80BFc

**Image**

![hypr-issues](https://user-images.githubusercontent.com/36679406/202116295-cfd8b1ee-2483-4f25-bf6f-2ae59b1d9f6e.png)

**Log:**
Please attach a log. (paste it into pastebin and paste here the url) The log can be found in a temp file located in /tmp/hypr/hypr.log.

[hypr.log](https://github.com/hyprwm/Hypr/files/10019058/hypr.log)

**Comment by vaxerski** [comment]
This is my config that has worked under Hypr back in the day, see the bottom of the file for workspaces.

[user_modules.txt](https://github.com/hyprwm/Hypr/files/10020931/user_modules.txt)

TLDR: I think it's because it should be
```ini
[module/workspaces]
type = internal/xworkspaces
```

**Comment by arcolinuxz** [comment]
your file does not include workspaces or ewmh

**Comment by vaxerski** [comment]
sorry, wrong file. I am trippin.
[modules.txt](https://github.com/hyprwm/Hypr/files/10021688/modules.txt)

**Comment by arcolinuxz** [comment]
it says we can see the desktops with 

`xprop -root _NET_DESKTOP_NAMES`

the message is  that this variabel is not found

Copy/paste your code over into my config

**Comment by vaxerski** [comment]
[polybar.tar.gz](https://github.com/hyprwm/Hypr/files/10022099/polybar.tar.gz)

> the message is that this variabel is not found

that's... odd...

**Comment by arcolinuxz** [comment]
This one is now solved

**Comment by arcolinuxz** [comment]
Sometimes I do see them.

Like here
I see them double - I can open a new workspace as well 
I can even move windows to the other workspace...

![image](https://user-images.githubusercontent.com/36679406/202713562-344529ab-582d-4e9b-b65e-528e2fc8eb40.png)

**Comment by vaxerski** [comment]
genuinely no clue, haven't experienced that myself. Just tested and they seem to work A-OK on my system.
